### ISSUE TITLE
Manifest errors, pack version 1.6.4.7

### ISSUE THREAD

**Issue by kinnison** [breakinblocks/nutsandboltstorqued#204]
The pack depends on ElecCore but the listed fileID may have been typo'd.

The manifest calls for file 2408169 but the closest file ID matching version 1.10.2 of minecraft is 2408869
(typo of 8 -> 1)

Adding that version of ElecCore then exposes that thermal foundation/expansion aren't downloaded:

Thermal Foundation lists a fileid I can't find at all: (2396623)

Thermal Expansion lists a fileid I can't find at all: (2396369)

Adding the closest versions in terms of fileid number for 1.10.2 that I can find bizarrely tells me the jars want 1.12,  which is odd because the filenames say 1.10.2.  I tried the latest 1.10.2 of each of those and it failed too.

Basically it seems impossible to get the pack off the ground currently.  Note, I was using CMPDL because I use Linux, so if this is all my fault, I'd appreciate guidance on getting the pack up.

Thanks in advance,

Daniel.

**Comment by Doomgull** [comment]
Sounds like a cmpdl issur, tbh, because curse generates the manifest.

**Comment by kinnison** [comment]
The manifest in the zip you download from curse is broken then.  Perhaps it's a transient fault on the curse site.  I'll keep an eye out and try again another time.

**Comment by Doomgull** [comment]
I know Vazkii updated CMPDL this last week, so perhaps there's something there.

 Have you tried using PortableJim's downloader to see if it's the same issue? Would at least help narrow down cause.

https://github.com/portablejim/curseDownloader
